src/app/components/bannerSection/bannerSection.module.css

```diff
@@ -1,6 +1,6 @@
 .bannerSection {
   background-color: #0f1038;
-  width: 100%;
+  width: 102%;
```

Comment:
The fix for this issue is not with the width being over 100%. If you look at your screenshot, even the blue rectangle above is off from the right side as well. The page has some margin being added on the right that needs to be removed.

Reply:
Okay understood, i am working on it

Reply:
![Screenshot from 2024-03-28 17-58-16](https://github.com/dallassoftwaredevelopers/DSDsite/assets/77767745/977eb11c-c222-4923-8a81-6100f1d25530)
I have removed the extra margin on the right side, is this correct? And do i need to make new PR because i have made this change on heroSection.module.css file

Reply:
No need for a new PR, just commit your changes.

Reply:
![image](https://github.com/dallassoftwaredevelopers/DSDsite/assets/39189903/1bc69ba4-ef7a-4337-a141-6023587b8813)
You need to remove this. it is still in your commit. After removing you need to verify that your change still works.

Reply:
Adding one more comment here, this still needs to be removed and after removing this you need to verify that the code still works.